deploy/determined_deploy/aws/cli.py

```diff
@@ -167,6 +167,16 @@ def make_up_subparser(subparsers: argparse._SubParsersAction) -> None:
         action="store_true",
         help="print deployment template",
     )
+    subparser.add_argument(
+        "--cpu-image",
```

Comment:
I might change the name to make it clearer that we are talking about the environment image and not the agent image

Reply:
Change all reference to cpu-env-image / gpu-env-image / equivalent.